library/src/main/java/com/devbrackets/android/exomedia/ui/widget/VideoControls.java

```diff
@@ -140,25 +165,20 @@
      */
     protected abstract void updateTextContainerVisibility();
 
-    public VideoControls(Context context) {
-        super(context);
-        setup(context);
-    }
-
-    public VideoControls(Context context, AttributeSet attrs) {
-        super(context, attrs);
-        setup(context);
-    }
-
-    public VideoControls(Context context, AttributeSet attrs, int defStyleAttr) {
-        super(context, attrs, defStyleAttr);
-        setup(context);
-    }
+    /***
+     * Updates the current timestamp
+     *
+     * @param position The position in milliseconds
+     */
+    protected void updateCurrentTime(long position) {
+        // optimization :
+        // update the timestamp text per second regarding the 'reset' or 'seek' operations.
+        if ((currentTimeTextView.getVisibility() == VISIBLE) &&
```

Comment:
This will cause a delay (up to a second) in updating the timestamp after the controls are shown again, please remove the `VISIBLE` check

Reply:
Thanks. Already removed the check.